packages/vscode-extension/src/telemetry/extTelemetryEvents.ts

```diff
@@ -79,6 +79,8 @@ export enum TelemetryEvent {
   DebugNpmInstall = "debug-npm-install",
 
   Survey = "survey",
+
+  EditSecret = "edit-secret",
```

Comment:
Do you need another event to indicate whether edit secret is successful or not? You can see from the event file that the common pattern is event-name-start for start and event-name with property success(yes/no) to indicate the result

Reply:
Thanks, updated.